src/quo2/components/tags/network_tags/view.cljs

```diff
@@ -0,0 +1,83 @@
+(ns quo2.components.tags.network-tags.view
+  (:require [quo2.components.tags.base-tag :as base-tag]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.tags.network-tags.style :as style]
+            [react-native.core :as rn]))
+
+(defn outer-resource-container
+  [size background-color]
+  {:background-color background-color
+   :border-radius    size
+   :width            size
+   :height           size
+   :margin-left      (case size
+                       32 -12
+                       24 -8)
+   :align-items      :center
+   :justify-content  :center})
+
+(defn network-group
+  []
+  (fn [{:keys [network-icon size background-color]
+        :or   {size 24}}]
+    [rn/view
+     {:style (style/network-tags-child)}
+     ^{:key network-icon}
+     [rn/view {:flex-direction :row}
+      [rn/view (outer-resource-container size background-color)
+       [rn/image
+        {:source network-icon
+         :style  {:height        (case size
+                                   32 28
+                                   24 20)
+                  :width         (case size
+                                   32 28
+                                   24 20)
+                  :border-radius size}}]]]
+    ]))
+
+(defn selected-group-count
+  [networks]
+  (cond
+    (> (count networks) 3) 3
+    :else
+    (count networks)))
+
+(defn tag-networks
+  []
+  (fn [{:keys [networks size background-color]
+        :or   {size 24}}]
+    (let [selected-groups (take (selected-group-count networks) networks)]
+      [rn/view
+       {:style (style/network-tags-child)}
+       (for [{:keys [network-icon id]} selected-groups]
+         ^{:key id}
+         [network-group
+          {:network-icon     network-icon
+           :size             size
+           :background-color background-color}])])))
+
+(defn network-tags
+  [_ _]
+  (fn [{:keys [title networks size background-color on-press]
+        :or   {size 24}}]
+    [rn/view
+     {:style (style/network-tags-container size)}
+     [base-tag/base-tag
```

Comment:
Here you have implemented too much I think ->
if you look at the designs you can see it's using another of our components which we already have implemented ->
<img width="423" alt="Screenshot 2023-07-24 at 09 40 44" src="https://github.com/status-im/status-mobile/assets/22799766/d76a4cbb-f8ec-4584-9947-6e0666065d92">

feel free to reach out on discord and we can discuss 👍